src/app/components/elements/panels/SelectedEventDetails.tsx

```diff
@@ -79,9 +108,13 @@ export const SelectedEventDetails = ({ eventId }: { eventId: string }) => {
             <strong>Event Date & Time: </strong>
             <DateString>{selectedEvent.date}</DateString> - <DateString>{selectedEvent.endDate}</DateString>
             <br />
-            <strong>Booking deadline: </strong>
-            <DateString>{selectedEvent.bookingDeadline}</DateString>
-            <br />
+            {selectedEvent.bookingDeadline && (
+              <>
+                <strong>Booking deadline: </strong>
+                <DateString>{selectedEvent.bookingDeadline}</DateString>
+                <br />
```

Comment:
I know you're doing this because we were already using `<br />` in this file, but if there is a quick and easy way to do this with CSS, it would be good to refactor the usages of `<br />` in this file.

Reply:
Yeah I nearly refactored this one yesterday, but then I figured the `<br />` is literally doing the job it's meant to do and breaking up a block of text. I've now changed these to paragraphs instead.